aries-site/src/pages/templates/user-feedback-collection.mdx

```diff
@@ -0,0 +1,142 @@
+perceptions around the ease-of-use and/or satisfaction with particular content, features, 
+or workflows.
+
+
+Feedback may be [actively solicited](#solicited-feedback) or [passively collected](#capturing-unsolicited-feedback).
+
+### When not to use
+Don’t collect feedback in the following use cases:
+- If it disrupts a user flow or is intrusive in any way.
+- There is no actionable plan for what to do with the data.
+- In locations where users do not have enough context to form a valid opinion. 
+- There are too many instances soliciting feedback which can [dilute feedback effectiveness](#sampling-frequency).
+
+### Sampling frequency
+Avoid over sampling and/or diluting effectiveness by limiting who is asked for feedback and 
+how often. Consider implementing "rules of engagement" specifying under what conditions an 
+individual user should be solicited for feedback. Example engagement rules might be:
+- Has not submitted response to this questionnaire in the last 90 days.
+- Has not been asked to complete this questionnaire in the last 30 days.
+- Has not been asked to complete any questionnaire in the last 7 days.
+
+## Anatomy
+<Example plain>
+  <FeedbackAnatomy />
+</Example>
+
+| Label  | Region       | Purpose                                                              |                  Required                   | Notes                                                                                                      |
+| ------ | ------------ | -------------------------------------------------------------------- | :-----------------------------------------: | ---------------------------------------------------------------------------------------------------------- |
+| **1**  | **Title**    | Title prompting user for their feedback.                             | <Checkmark a11yTitle="true" size="small" /> | <span>Use the [semantically correct heading level](/foundation/typography?q=semantically#semantic-usage-of-heading-levels) for the title. "We'd love your feedback" is a nice, appreciative default title.</span>       |
+| **2**  | **Body**     | The body may be populated with as many questions as appropriate.          | <Checkmark a11yTitle="true" size="small" /> | <span>Question types can include [StarRating](https://v2.grommet.io/starrating), [ThumbsRating](https://v2.grommet.io/thumbsrating), select all that apply (e.g. checkbox group), choose one (e.g. RadioButtonGroup), open ended (e.g. TextArea), or any other input types.</span> |                                                             
+| **3**  | **Footer **  | Contains two actions: "Submit" and "Dismiss" if used in Layer modal. Otherwise only "Submit" is needed inline. | <Checkmark color='text-strong' a11yTitle="true" size="small" /> |  Submit button sends survey responses to the HPE customer experience team and initiates thank you message. If presented in a modal, the Dismiss button allows the user to bypass the survey and continue their product experience. |
```

Comment:
```suggestion
| **3**  | **Footer **  | Contains two actions: "Submit" and "Dismiss" if used in Layer modal. Otherwise only "Submit" is needed inline. | <Checkmark color='text-strong' a11yTitle="true" size="small" /> |  Submit button sends survey responses to the HPE Customer Experience team and initiates thank you message. If presented in a modal, the Dismiss button allows the user to bypass the survey and continue their product experience. |
```